app/client/components/HelpCentrePage.tsx

```diff
@@ -41,25 +41,24 @@ const HelpCentreRouter = () => {
     <Main>
       <Global styles={css(`${global}`)} />
       <Global styles={css(`${fonts}`)} />
-      <WithLiveChatContainer>
-        <Suspense fallback={<HelpCentrePageSkeleton />}>
-          <Router primary={true} css={{ height: "100%" }}>
-            <HelpCentre path="/help-centre" />
+      <WithLiveChatContainer />
```

Comment:
I suggest renaming this component to something like `LiveChat` and it now represents the LiveChat itself, not a container that wraps its children with the Live Chat.